packages/cos/collection.yaml

```diff
@@ -2,20 +2,18 @@ packages:
   - &cos
     name: "cos"
     category: "system"
-    version: 0.7.14-2
+    version: 0.7.23
```

Comment:
master version is 0.7.15

Reply:
current requires of cos has the installer at version 0.27.0, you would need to bump it if you bump the installer version also https://github.com/rancher-sandbox/cOS-toolkit/blob/master/packages/cos/build.yaml#L11